### ISSUE TITLE
not able to configure logical_backup_docker_image for postgres-operator:v1.4.0

### ISSUE THREAD

**Issue by andypeng2015** [zalando/postgres-operator#1712]
Please, answer some short questions which should help us to understand your problem / question better?

- **Which image of the operator are you using?**
registry.opensource.zalan.do/acid/postgres-operator:v1.4.0
- **Where do you run it - cloud or metal? Kubernetes or OpenShift?**
 K8s
- **Are you running Postgres Operator in production?** 
yes
- **Type of issue?** 
question

I encountered `/dump.sh: line 56: LOGICAL_BACKUP_PROVIDER: unbound variable` and changed logical_backup_docker_image manually to  "registry.opensource.zalan.do/acid/logical-backup:master-58" solved the issue.

So I went to add `
"logical_backup_docker_image": "registry.opensource.zalan.do/acid/logical-backup:master-58"`
into postgres-operator configmap, but it still use default value in log `"LogicalBackupDockerImage\": \"registry.opensource.zalan.do/acid/logical-backup\"`

my question is How to add `logical_backup_docker_image` so operator v1.4.0 can load it?

postgres-operator image is  'registry.opensource.zalan.do/acid/postgres-operator:v1.4.0'

the other 2 configs FYI
"connection_pooler_image": "registry.opensource.zalan.do/acid/pgbouncer:master-6",
"docker_image": "registry.opensource.zalan.do/acid/spilo-12:1.6-p2",

**Comment by FxKu** [comment]
you upgraded the configmap + deleted the operator pod so that the new config is loaded?

**Comment by andypeng2015** [comment]
> You upgraded the configmap + deleted the operator pod so that the new config is loaded? And of course, consider upgrading.

yes, I updated the configmap and deleted the operator pod, but still not working, the pod changed the configmap to original value

@FxKu **I am considering upgrade from 1.4.0 to 1.5.0, can I do the upgrade in place(I mean just change the docker image version to the listed below and also change config for the breaking changes below)? or I need to create a new cluster with all the new config? any impact to existing backup in S3? can I rollback to 1.4.0 if upgrading fails?**



https://github.com/zalando/postgres-operator/releases/tag/v1.5.0

Docker image: registry.opensource.zalan.do/acid/postgres-operator:v1.5.0
UI Docker image: registry.opensource.zalan.do/acid/postgres-operator-ui:v1.5.0-dirty
Sample logical backup image: registry.opensource.zalan.do/acid/logical-backup:master-58
Connection pooler image: registry.opensource.zalan.do/acid/pgbouncer:master-7

Breaking changes
    Update your postgres-operator cluster role (deployments and events added) (#799, #952)
    Passing ClusterLabels as JSON to Spilo (#877)
    Switching to Go 1.14 (#882)
    Scalyr sidecar deprecated (#945, #890)

**Comment by FxKu** [comment]
Upgrading the operator is described [here](https://github.com/zalando/postgres-operator/blob/master/docs/administrator.md#upgrading-the-operator). Why 1.5.0 and not straight to the latest release. Keep in mind the operator is just there for high level management tasks and not about Postgres DBs that are running in your cluster.

Make sure, to update CRD first, then update configuration if needed and then replace docker image in operator deployments. And of course do this in a test environment first and then in production.

**Comment by andypeng2015** [comment]
> Upgrading the operator is described [here](https://github.com/zalando/postgres-operator/blob/master/docs/administrator.md#upgrading-the-operator). Why 1.5.0 and not straight to the latest release. Keep in mind the operator is just there for high level management tasks and not about Postgres DBs that are running in your cluster.

thanks, just want to upgrade to the closest version to 1.4.0 with the fix for cron job so 1.5.0 is chosen

> And regarding logical backup not working some more details (logs) would be helpful. Did you also remove the existing cron job?

I did not remove the existing cron job, just updated the configmap and deleted the operator pod, is it required? if so I will try again
